### DISCUSSION TITLE
JK_B2A20S20P  V11.XW, V11.26H  what ymal can I use?

### DISCUSSION THREAD

**Discussion by Alen0912** [syssi/esphome-jk-bms#624] [General]
I Have JK_B2A20S20P  V11.XW, V11.26H in 8S configuration, what ymal can i use?
Wired connection only. I have ESP32-Wroom-32D

**Comment by alanv72** [comment]
https://github.com/syssi/esphome-jk-bms/blob/main/esp32-example.yaml

This one is probably for you. Make sure you map the correct GPIO pins. Sometimes different knock-off ESP32 switch them up.

I am using a nodemcuv4 ESP8266 over RS485 UART with good results. There are a few limitations of the UART communications. You can't control the balancing. Which is fine for me, I want the BMS to control when balancing kicks in. The drilldown details of each cell is also not available (ie. each min/max cell voltages and resistance).

If you need these the bluetooth connection has more (ble). Go figure... ¯\_(ツ)_/¯
